hive-llap/README.md

```diff
@@ -0,0 +1,143 @@
+
+# Hive LLAP Initialization Action
+This initialization action configures Google Cloud Dataproc to run Hive LLAP. For more information on Hive LLAP please visit: https://cwiki.apache.org/confluence/display/Hive/LLAP
+
+## Optional Configuration Options
+* num_llap_nodes: The number of nodes you wish LLAP to run on. This configuration will use 100% of the YARN resources LLAP is deployed on. Do not deploy on 100% of the deployed dataproc workers. You need available nodes in order to run the Tez Application Masters spawned by Hive sessions. Understand what level of concurrency you require to figure out how many workers you need available to support your users.
```

Comment:
Can we add some link that user can go through to understand the level of concurrency required?

Reply:
fixed